plasmapy/particles/tests/test_particle_class.py

```diff
@@ -947,12 +949,16 @@ def test_customized_particles(cls, kwargs, attr, expected):
     (CustomParticle, {"charge": np.complex128(5 + 2j)}, InvalidParticleError),
     (CustomParticle, {"mass": np.complex128(5 + 2j)}, InvalidParticleError),
     (CustomParticle, {"mass": -1e-36 * u.kg}, InvalidParticleError),
+    (CustomParticle, {"mass": "not a mass"}, InvalidParticleError),
+    (CustomParticle, {"mass": "5.0 km"}, InvalidParticleError),
```

Comment:
I mean, with `c = 1` that might just make sense...

Reply:
Good old natural units! This was not intentional. I use to be so good w/ them but now I can barely remember them.